store/src/java/com/zimbra/cs/service/mail/SendMsg.java

```diff
@@ -132,8 +132,11 @@ public Element handle(Element request, Map<String, Object> context, QName respQn
                boolean noSaveToSent = request.getAttributeBool(MailConstants.A_NO_SAVE_TO_SENT, false);
                boolean fetchSavedMsg = request.getAttributeBool(MailConstants.A_FETCH_SAVED_MSG, false);
                boolean deliveryReport = false;
-               if (LC.delivery_report_enabled.booleanValue()) {
-                   deliveryReport = request.getAttributeBool(MailConstants.A_DELIVERY_RECEIPT_NOTIFICATION, false);
+               String deliveryStatusNotificationEnabled = authAcct.getAttr(Provisioning.A_zimbraFeatureDeliveryStatusNotificationEnabled);
+               if (deliveryStatusNotificationEnabled != null) {
```

Comment:
combine both if conditions into one

Reply:
Removing the null check as it gets taken care of in the equals condition.